core/src/test/scala/org/apache/spark/scheduler/SchedulerIntegrationSuite.scala

```diff
@@ -27,6 +27,8 @@ import scala.language.existentials
 import scala.reflect.ClassTag
 
 import org.scalactic.TripleEquals
+import org.scalatest.concurrent.Eventually._
+import org.scalatest.time.SpanSugar._
 import org.scalatest.Assertions.AssertionsHelper
```

Comment:
derp sorry just noticed this one is out of order (should be above org.scalatest.concurrent)